app/actions/ModActionPanel/QuickAction.tsx

```diff
@@ -669,9 +669,13 @@ function Form(
                       id="labels"
                       name="labels"
                       formId={FORM_ID}
-                      defaultLabels={currentLabels.filter(
-                        (label) => !isSelfLabel(label),
-                      )}
+                      defaultLabels={currentLabels.filter((label) => {
+                        const serviceDid = client.getServiceDid()
```

Comment:
Is it possible that this "DID" contains the service id `#atproto_labeler`, yet `l.src` below will omit that?  Or does `getServiceDid()` return just the DID without the service id on the end?